iree/compiler/Dialect/HAL/Transforms/Passes.cpp

```diff
@@ -38,7 +38,7 @@ struct TransformOptions : public PassPipelineOptions<TransformOptions> {
   Option<bool> linkExecutables{
       *this, "link-executables",
       llvm::cl::desc("Whether to link hal.executable ops together."),
-      llvm::cl::init(true)};
+      llvm::cl::init(false)};
```

Comment:
This will cause test failures in tests that don't explicitly set the flag value, such as https://github.com/google/iree/blob/main/iree/compiler/Dialect/HAL/Target/VMLA/test/smoketest.mlir and https://github.com/google/iree/blob/735f3e0642beb03e95f1bc15ee7ade7c8df1ef06/iree/compiler/Dialect/HAL/Target/LLVM/test/binary_op.mlir#L14 (the 2nd at least is over-constrained :P)

I'm okay with continuing without updating the tests, just another thing to resolve as part of getting this branch eventually stable enough to merge into `main` (either by fixing linking or updating the tests).

Reply:
Ugh yeah - will see if I can't figure out a good fix as part of this branch so we don't regress on main.